Automattic/simplenote-electron
# Adds Stylelint for sass linting

### Fix

Closes https://github.com/Automattic/simplenote-electron/issues/1158

This PR adds stylelint and prettier standard configs with some exceptions for sass.

- I removed the extra whitespace configurations which I believe would be at odds with our prettier configuration.
- I removed [font: inherit ](https://github.com/Automattic/simplenote-electron/compare/develop...add/stylelint#diff-77be6b3031462c4580c10faa0d40ff36L258) from the normalize file as it is invalid CSS. 
- I fixed all the errors where we had no consistent style.
- Removed or added empty lines as required.
- Removed duplicate selectors.

### Test
1. Run app, do fonts look okay
2. Make a change to the css that stylelint will catch
3. Run linting, does it catch the error?
4. Run a commit, if there is an error you should not be able to commit
